src/main/stackpack/resources/RELEASE.md

```diff
@@ -1,5 +1,11 @@
 # Splunk StackPack Release Notes
 
+## 1.2.0 (TBD)
```

Comment:
this version needs to be bumper in stackstate aswell, and the time should be set